lib/Dialect/Torch/IR/TorchOps.cpp

```diff
@@ -3409,25 +3439,25 @@ OpFoldResult AtenItemOp::fold(FoldAdaptor adaptor) {
 OpFoldResult AtenOnesOp::fold(FoldAdaptor adaptor) {
   SmallVector<int64_t> sizes;
   if (!matchPattern(getSize(), m_TorchListOfConstantInts(sizes))) {
-    LLVM_DEBUG(llvm::dbgs() << "Failing to fold AtenOnesOp: size operand is "
```

Comment:
I've been wondering. Why do we not see more of these debug statements.

Since it's using the LLVM_DEBUG macro, they doesn't make it into non debug builds anyway.

Reply:
You want to be strategic where you put them. Too many messages and you fill the debug logs with garbage. In this case every time you run `canonicalize` one of these would print for every matching operation (as you print the bail out). This would make debugging compilation runs of ever moderately sized test cases basically unusable. So while we can include them per pass, its usually bad per rewriter / folder.